source/Generic/PlayNotes/PlayNotesSettings.cs

```diff
@@ -85,6 +85,28 @@ public PlayNotesSettingsViewModel(PlayNotes plugin)
                 imageStyle.Setters.Add(new Setter(Image.MaxWidthProperty, maxWidthBinding));
             }
 
+            if (ResourceProvider.GetResource(baseStyleName) is Style baseStyle &&
```

Comment:
Did you build and test the changes? Correct me if I'm wrong but `baseStyleName` is not defined anywhere in this scope

Reply:
Forgot the line, sorry, fixed.

Reply:
I see, no worries. Thanks for contributing.